styles/layouts/scss/header.scss

```diff
@@ -1,7 +1,7 @@
 @use '../../global-styles/_colors.scss' as *;
 @use '../../global-styles/_fonts.scss' as *;
 @use '../../global-styles/_screens.scss' as *;
-@import '../../global-styles/_key-frames.scss';
+@use '../../global-styles/_key-frames.scss' as *;
 
```

Comment:
.

Reply:
oops the animation worked fine in the project , this is why i haven't noticed it. how did you detect it ? :)

Reply:
I have just noticed a log in the Sass compiler console :)